ttnn/cpp/ttnn/operations/conv/conv2d/device/conv2d_op.cpp

```diff
@@ -160,16 +151,10 @@ std::vector<tt::tt_metal::LegacyShape> OptimizedConvNew::compute_output_shapes(c
     auto padded_shape_c = tt::round_up(this->output_channels, TILE_WIDTH);
     auto output_padding = Padding(
         {{0, 0}, {0, 0}, {0, (padded_shape_w - shape_w)}, {0, (padded_shape_c - shape_c)}}, Padding::PadValue::Zero);
-    auto output_tensor_shape = ttnn::Shape(tt::tt_metal::LegacyShape({1, 1, padded_shape_w, padded_shape_c}, output_padding));
-    return {output_tensor_shape.value};
-}
+    auto output_shape = tt::tt_metal::LegacyShape({1, 1, padded_shape_w, padded_shape_c}, output_padding);
```

Comment:
removal of the use of LegacyShape is on the OP owners?

Reply:
Yes. Removal of Shape/LegacyShape within each individual OP is on the OP's owners. It may involve modifying the operation to avoid unnecessary paddings.